src/itemlistformaction.cpp

```diff
@@ -1040,10 +1040,22 @@ std::string ItemListFormAction::item2formatted_line(const ItemPtrPosPair& item,
 	fmt.register_fmt('n', item.first->unread() ? "N" : " ");
 	fmt.register_fmt('d', item.first->deleted() ? "D" : " ");
 	fmt.register_fmt('F', item.first->flags());
-	fmt.register_fmt('D',
-		utils::mt_strf_localtime(
-			datetime_format,
-			item.first->pubDate_timestamp()));
+
+	const auto is_found = datetime_format.find("%L");
+	if (is_found != std::string::npos) {
+		auto article_epoch = std::stol(
+				utils::mt_strf_localtime("%s", item.first->pubDate_timestamp()));
+		auto current_epoch = std::chrono::duration_cast<std::chrono::seconds>(
+				std::chrono::system_clock::now().time_since_epoch()).count();
+		auto days = (current_epoch - article_epoch) / 86400;
```

Comment:
This block is okay, but I think it can be simplified if we go all-in on `std::chrono`:

1. `RssItem::pubDate_timestamp()` returns `time_t`, which we can convert into a `time_point` [directly](https://en.cppreference.com/w/cpp/chrono/system_clock/from_time_t);
2. the above removes the need to convert `current_epoch` into a number — remove `count()` to keep the result a `time_point`;
3. replace `/ 86400` by a cast to `std::chrono::days`;
4. to make the result more readable, add `using namespace std::chrono` to the top of the block, and drop the quantification from names of that namespace.

Also, all of the variables here never change, so please mark them with `const`.